rust/edgehog-device-forwarder-proto/src/proto.rs

```diff
@@ -0,0 +1,112 @@
+#[derive(Eq)]
```

Comment:
Is it correct that this file has already been added? It should be generated after the pull_request...

Reply:
The file was generated manually in the commit before the CI one. It's the same as the one that would be generated by the CI and would be overwritten. If you want I can still remove it.